v19.2/insert.md

```diff
@@ -67,68 +67,81 @@ key. Using `ON CONFLICT` is therefore more flexible.
 
 ## Examples
 
-All of the examples below assume you've already created a table `accounts`:
-
-{% include copy-clipboard.html %}
-~~~ sql
-> CREATE TABLE accounts(
-    id INT DEFAULT unique_rowid() PRIMARY KEY,
-    balance DECIMAL
-);
-~~~
+{% include {{page.version.version}}/sql/movr-statements.md %}
 
 ### Insert a single row
 
 {% include copy-clipboard.html %}
 ~~~ sql
-> INSERT INTO accounts (balance, id) VALUES (10000.50, 1);
+> INSERT INTO users VALUES
+    ('c28f5c28-f5c2-4000-8000-000000000026', 'new york', 'Petee', '101 5th Ave', '1234567890');
 ~~~
 
 {% include copy-clipboard.html %}
 ~~~ sql
-> SELECT * FROM accounts;
+> SELECT * FROM users WHERE city='new york';
 ~~~
 
 ~~~
-  id | balance
-+----+----------+
-   1 | 10000.50
-(1 row)
+                   id                  |   city   |       name       |           address           | credit_card
++--------------------------------------+----------+------------------+-----------------------------+-------------+
+  00000000-0000-4000-8000-000000000000 | new york | Robert Murphy    | 99176 Anderson Mills        | 8885705228
+  051eb851-eb85-4ec0-8000-000000000001 | new york | James Hamilton   | 73488 Sydney Ports Suite 57 | 8340905892
+  0a3d70a3-d70a-4d80-8000-000000000002 | new york | Judy White       | 18580 Rosario Ville Apt. 61 | 2597958636
+  0f5c28f5-c28f-4c00-8000-000000000003 | new york | Devin Jordan     | 81127 Angela Ferry Apt. 8   | 5614075234
+  147ae147-ae14-4b00-8000-000000000004 | new york | Catherine Nelson | 1149 Lee Alley              | 0792553487
+  c28f5c28-f5c2-4000-8000-000000000026 | new york | Petee            | 101 5th Ave                 | 1234567890
```

Comment:
Movr comment, not docs comment: we should update the credit card handling to be more realistic at some point, hashed, elliptic curved, or "just use Stripe", or whatever the correct behavior should be - not plaintext.

Reply:
Agreed. I'll add this to the notes.